templates/user/edit-profile.html

```diff
@@ -177,6 +196,65 @@
             });
         });
     </script>
+    <script src="{{ static('libs/clipboard/clipboard.js') }}"></script>
+    <script src="{{ static('libs/clipboard/tooltip.js') }}"></script>
+    <script type="text/javascript">
+        $(function () {
+            $('#generate-scratch-codes-button').click(function (event) {
```

Comment:
```suggestion
            $('#generate-scratch-codes-button').click(function(event) {
```

Nit: space comes after `function` iff there are no arguments to the function, i.e. the param list is `()`.